src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitDraftTest.java

```diff
@@ -0,0 +1,100 @@
+import static io.restassured.RestAssured.useRelaxedHTTPSValidation;
+import static org.hamcrest.Matchers.isEmptyOrNullString;
+import static org.hamcrest.Matchers.not;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import java.util.Base64;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.domain.wrapper.SyaBenefitType;
+import uk.gov.hmcts.reform.sscs.domain.wrapper.SyaCaseWrapper;
+import uk.gov.hmcts.reform.sscs.idam.Authorize;
+import uk.gov.hmcts.reform.sscs.idam.IdamApiClient;
+import uk.gov.hmcts.reform.sscs.util.SyaServiceHelper;
+
+@RunWith(SpringRunner.class)
+@TestPropertySource(locations = "classpath:config/application_e2e.properties")
+@SpringBootTest
+public class SubmitDraftTest {
+
+    @Value("${test-url}")
+    private String testUrl;
+
+    @Autowired
+    private IdamApiClient idamApiClient;
+
+    @Value("${idam.oauth2.client.secret}")
+    private String idamOauth2ClientSecret;
+
+    @Value("${idam.oauth2.redirectUrl}")
+    private String idamOauth2RedirectUrl;
+
+    @Value("${idam.oauth2.citizen.email}")
+    private String username;
+
+    @Value("${idam.oauth2.citizen.password}")
+    private String password;
+
+    private String userToken;
+
+    @Before
+    public void setUp() {
+        baseURI = testUrl;
+        useRelaxedHTTPSValidation();
+        userToken = getIdamOauth2Token(username, password);
+    }
+
+    @Test
+    public void givenDraft_shouldBeStoredInCcd() {
+        SyaCaseWrapper draftAppeal = new SyaCaseWrapper();
+        draftAppeal.setBenefitType(new SyaBenefitType("PIP", "pip benefit"));
+
+        RestAssured.given()
+            .log().method().log().headers().log().uri().log().body(true)
+            .contentType(ContentType.JSON)
+            .header(new Header(AUTHORIZATION, userToken))
+            .body(SyaServiceHelper.asJsonString(draftAppeal))
+            .post("/drafts")
+            .then()
+            .statusCode(HttpStatus.CREATED_201)
+            .assertThat().body("id", not(isEmptyOrNullString())).log().all(true);
+    }
+
+    public String getIdamOauth2Token(String username, String password) {
+        String authorisation = username + ":" + password;
+        String base64Authorisation = Base64.getEncoder().encodeToString(authorisation.getBytes());
+
+        Authorize authorize = idamApiClient.authorizeCodeType(
+            "Basic " + base64Authorisation,
+            "code",
+            "sscs",
```

Comment:
Make this a constant so that it is obvious what it is and not just a `magic string`.

Reply:
done